examples/gno.land/p/demo/users/types.gno

```diff
@@ -1,7 +1,5 @@
 package users
 
-import "std"
-
 type AddressOrName string
 
 func (aon AddressOrName) IsName() bool {
```

Comment:
These two methods also seem very application specific.  I don't think there is any obvious reason to prefix usernames with `@` in general.  Though I do think this type can still be helpful to differentiate a string between being an address or name.  This might be a good opportunity to make each method accept a `usernamePrefix string` argument that allows the application to decide this.

Reply:
@deelawn I think the way it currently is makes a lot of sense!

r/demo/users is, to me, the "username" realm every other realm should use to identify users. p/demo/users is not meant to be the "universal" implementation for all user systems.

If we have a realm function which is, say, the following:

```go
func SendMessage(dst users.AddressOrName, title, content string)
```

having users.AddressOrName as the type indicates that, unless the realm's doing something weird, the function will accept a full address or also a username prefixed with `@`. So I'm personally in favour to keep it this way; `p/demo/users` is allowed to have some opinions :)